docs/source/metric_integrations.mdx

```diff
@@ -0,0 +1,136 @@
+# Using 🤗 Evaluate in other frameworks
+
+This guide outlines how `evaluate` metrics can be used with popular machine learning frameworks.
+
+## 🤗 Transformers
```

Comment:
WDYT Of having separate pages for each library + a general one? Similar to https://huggingface.co/docs/hub/transformers. Then anyone could come and create a new page for their own libraries.